x-pack/plugins/ml/public/application/timeseriesexplorer/components/forecasting_modal/forecasting_modal.js

```diff
@@ -178,8 +179,21 @@ export class ForecastingModalUI extends Component {
   runForecastErrorHandler = (resp, closeJob) => {
     this.setState({ forecastProgress: PROGRESS_STATES.ERROR });
     console.log('Time series forecast modal - error running forecast:', resp);
-    if (resp && resp.message) {
-      this.addMessage(resp.message, MESSAGE_LEVEL.ERROR, true);
+
+    let errorMessage;
+    if (resp) {
+      errorMessage = extractErrorMessage(resp);
+    }
```

Comment:
```suggestion
const errorMessage = resp ? extractErrorMessage(resp) : undefined;
    
```

Reply:
Done in https://github.com/elastic/kibana/pull/139345/commits/c99a3bf4bb33011d3b2fd07f010bd5177f92805c